content/working_with/console/widgets/deploymentsView.md

```diff
@@ -0,0 +1,279 @@
+2. `csys-obj-type` - either `service` or `environment`
+   If there is no such label, the deployment is a service by default.
+
+Forming a hierarchical structure allows drilling down into deployments that are
+directly attached to a parent environment, as well as aggregating and
+propagating statuses of deployments for better problem discovery.
+
+### Deployment status
+
+A single deployment's status is:
+
+- _requires attention_ (red exclamation mark) if one of the following applies:
+  - at least one node is not successfully installed (and there is no workflow
+    in progress)
+  - the last workflow execution failed
+- _in progress_ (yellow spinner) if there is a workflow in progress
+- _good_ otherwise
+
+### Propagating deployment statuses
+
+If a deployment has child deployments, then their statuses are propagated so
+that:
+
+1. If there is at least nested deployment that requires attention, the parent's
+   status will be _requires attention_ too.
+2. Otherwise, if there is at least nested deployment that is in progress,
+   the parent's status will be _in progress_ too (unless the parent deployment
+   itself _requires attention_).
+
+### Drilling down to child deployments
+
+If a deployment has child deployments, the drill-down buttons in the top part of
+the right pane become active. Clicking them will navigate to a nested page that
+present the deployments (only environments or services, depending on the button
+clicked) from the next level in the hierarchy.
+
+The nested page has only a single Deployments View widget, but it can be
+customized using the
+[Edit Mode]({{< relref "working_with/console/customization/edit-mode.md" >}}).
+
+Drilling down can happen multiple times if there are more levels in the
+deployments hierarchy.
+
+To go back to the parent page, use the breadcrumbs in top part of the page.
+
+<!-- TODO: screenshot -->
+
+## Features
+
+### Filtering deployments
+
+Deployments visible in the table can be filtered in 3 ways:
+
+1. Typing in the search bar above the table
+2. Setting a default filter in the widget's configuration This filter will be
+   persisted and used when viewing the widget again.
+3. Applying a local filter
+
+   By clicking the "Filter" button in the top-right
+   corner of the widget, one can select a local filter. It will override the
+   default filter picked in the widget's configuration. The local filter will be
+   reset when viewing the widget again.
+
+### Deployments Map
+
+Clicking the _Map_ button in the top part of the widget will open an additional
+map section in the top part of the widget. The map presents deployments in the
+sites that are attached to them.
+
+<!-- TODO: screenshot map -->
+
+The height of the map section, as well as whether it should be open by default
+can be changed using the widget's configuration.
+
+The color of the marker representing a deployment depends on the deployment's
+state:
+
+1. Blue - _good_
```

Comment:
Oh, I see that we've got misalignment with Sites Map widget. I used green for indicating all good.
IMHO we can handle that in 6.1 within RD-1780.

Reply:
Good point :+1: Sure, sounds good to do it in RD-1780